resilience4j/datadog_checks/resilience4j/check.py

```diff
@@ -0,0 +1,21 @@
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheckV2
+
+from .metrics import METRIC_MAP
+
+
+class Resilience4jCheck(OpenMetricsBaseCheckV2):
+    __NAMESPACE__ = "resilience4j"
+    DEFAULT_METRIC_LIMIT = 0
+
+    def __init__(self, name, init_config, instances):
+        super(Resilience4jCheck, self).__init__(name, init_config, instances)
+
+    def get_default_config(self):
+        return {
+            "metrics": [METRIC_MAP],
+            "send_distribution_sums_as_monotonic": "true",
+            "send_distribution_counts_as_monotonic": "true",
```

Comment:
The `OpenMetricsBaseCheckV2` reports a service check [by default](https://github.com/DataDog/integrations-core/blob/master/openmetrics/datadog_checks/openmetrics/data/conf.yaml.example#L167), you don't have to do anything on that side.